lib/rubocop/cop/rspec/repeated_example_group_description.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated example group descriptions.
+      #
+      # @example
+      #
+      #    # bad
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # bad
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    describe 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    # bad
+      #    RSpec.describe 'Animal', 'dog' do
+      #      # example group
+      #    end
+      #
+      #    RSpec.describe 'Animal', 'cat' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    describe 'cool feature' do
+      #      # example group
+      #    end
+      #
+      #    describe 'another cool feature' do
+      #      # example group
+      #    end
+      #
+      #    # good
+      #    context 'when case x' do
+      #      # example group
+      #    end
+      #
+      #    context 'when another case' do
+      #      # example group
+      #    end
```

Comment:
Keeping in mind there's no big difference between `context`/`describe`/`RSpec.describe` I'm not sure if so many good examples are needed.